muikku/src/main/webapp/resources/theme-muikku/sass/_communicator.scss

```diff
@@ -87,12 +79,16 @@
     &.workspace{
 
       &:before{
-         // @extend .icon-books;
-         // extend breaks icon in firefox
          content: "\e61b";
```

Comment:
This needs to be removed as the icon is added directly to dust file as element class. Content was dirty fix for immediate problem.